### DISCUSSION TITLE
EarlyPrint to DraCor workflow ideas

### DISCUSSION THREAD

**Discussion by cmil** [dracor-org/epdracor#9] [General]
I have a few suggestions for the workflow of creating the EngDraCor corpus from the EarlyPrint TEI files that I would like to discuss a bit, because it might help to establish a standard workflow for future corpora.

1. I would suggest to track the source files in a separate repository (as opposed to, for instance, keeping them in a sub directory inside the corpus repo) for the following reasons:
   -  it would reduce the size of the corpus repo (which is an advantage when we load a repo into the database via a ZIP file from GitHub.)
   - if we generally assume that the source files come from an external repository in future cases this could also be a repo that is not under DraCor control
   - we could have multiple source repos for one DraCor corpus
2. I would name the source repo for EngDraCor `engdracor-earlyprint` rather than `engdracor-sources` because it would make more clear which project the sources originate from and, as a convention, it would better support the hypothetical multi-source scenario mentioned above.

**Comment by ingoboerner** [comment]
totally agree to 1.

ad 2.:
OK, I see your point, because we will have different repos as sources, as it is the case with "gerdracor". Using "-sources" would make it easier, to locate the repo in the dracor github group; a user would always know, that the sources to a repo could be retrieved by appending the "-sources" to the repo name; if we use different suffixes, e.g. "engdracor-earlyprint", "dracor-textgrid" ... we should at least add a file to the main corpus repository ("engdracor", "gerdracor"), in which we list the source repos:
```
{
... ,
"source-repositories" : [ 
    {"name": "engdracor-earlyprint", 
      "url" : "https:github...." }
    ]
}
```
